source/security/client-side-field-level-encryption-guide.txt

```diff
@@ -262,6 +262,24 @@ Additional Dependencies
              <https://pypi.org/project/pymongocrypt/>`_
            - Python wrapper for the ``libmongocrypt`` encryption library.
 
+   .. tab::
+      :tabid: csharp
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Dependency Name
+           - Description
+
+         * - x64 Support
+           - x64 support is required for CSFLE
```

Comment:
Is this a .NET setting? Maybe not necessary, but is there a link that might help explain it to a Jasmin persona or non-C# developer like me?